dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/environment/EnvironmentExtractionPipeline.scala

```diff
@@ -3,11 +3,20 @@ package org.broadinstitute.monster.dap.environment
 import org.broadinstitute.monster.common.{PipelineBuilder, ScioApp}
 import org.broadinstitute.monster.dap.common._
 
+import java.time.{OffsetDateTime, ZoneOffset}
+import java.time.format.DateTimeFormatter
+
 // Ignore IntelliJ, this is used to make the implicit parser compile.
 import Args._
 
+class EnvironmentExtractionFailException() extends scala.Exception {}
+
 object EnvironmentExtractionPipeline extends ScioApp[Args] {
 
+  val formatter = DateTimeFormatter.ofPattern("MMMyyyy")
+
+  val ENVEpoch = OffsetDateTime.of(2020, 1, 1, 0, 0, 0, 0, ZoneOffset.ofHours(-5))
```

Comment:
⛏️ I think everywhere else we are using `Environment`, not `ENV`, I think this should be consistent with the rest of the naming in this file.